front/src/dayview/Dayview.js

```diff
@@ -148,20 +148,35 @@ class Dayview extends Component {
             key={key}
             name={props.tracks[key].name}
             state={props.tracks[key].trackSupervision}
-	    notice={props.tracks[key].notice}
+	          notice={props.tracks[key].notice}
             //TODO final react routing
             to={"/trackview/"+props.date.toISOString()+"/" + props.tracks[key].name}
           />
         );
       }
-
+      if (items.length == 6) {
```

Comment:
Use switch-case

Reply:
Or better yet, use an array of these "xGrids" and loop over that array by doing something like var classNames = grids[items.length]